caim_base/templates/base/wrapper.html

```diff
@@ -17,7 +17,7 @@
     <link rel="apple-touch-icon" href="{% static 'img/caim-icon.png' %}">
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700;800;900&display=swap" rel="stylesheet">
```

Comment:
Just curious, does this reduce or increase the number of fonts downloaded

Reply:
It would increase the payload a bit, since we're downloading additional weights to use (like in this case, where the designs called for bold fonts).